src/types/tmdb/show.ts

```diff
@@ -111,6 +113,20 @@ export interface ShowData {
     }[];
     media_type: 'movie' | 'tv' | 'person';
     genre_ids?: number[];
+    seasons?:
+        | [
```

Comment:
Is this first pipe character supposed to be here?

Reply:
@Thenlie No, lint puts it there.

Reply:
Ah weird. We could do this, but just a difference in formatting.
```ts
export interface Season {
    air_date: string;
    episode_count: number;
    id: number;
    name: string;
    overview: string;
    poster_path: string;
    season_number: number;
    vote_average: number;
}

export interface ShowData {
    id: number;
    // ...
    seasons?: Season[] | null;
}
```